AngularJS/scss/dashboard.scss

```diff
@@ -0,0 +1,93 @@
+@mixin book-text($color, $fontSize) {
+  color: $color;
+  font-size: $fontSize;
+  line-height: $line-height-base;
+}
+
+%robotoRegular {
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 400;
+}
+
+%robotoItalic {
+  @extend %robotoRegular;
+  font-style: italic;
+}
+
+%robotoBold {
+  @extend %robotoRegular;
+  font-weight: 700;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  margin: 20px auto;
+  max-width: 700px;
+}
+
+.book-search {
+  align-items: center;
+  display: flex;
+  margin: 20px 0;
+}
+
+.search-input {
+  background-color: $wild-sand ;
+  @extend %robotoItalic;
+  height: 40px;
+  margin: 10px 0 10px 15px;
+  padding: 10px;
+  width: 240px;
+}
+
+.search-img {
+  background: $atlantis url('assets/search.svg') no-repeat center ;
+  height: 40px;
+  width: 40px;
+}
+
+.custom-select {
+  background-image: linear-gradient(45deg, transparent 50%, $dusty-gray 50%),
+  linear-gradient(135deg, $dusty-gray 50%, transparent 50%),
+  linear-gradient(to right, $wild-sand, $wild-sand);
```

Comment:
Why are you adding so much linear gradients?

Reply:
to create the arrow in the select